.vscode/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build with build.py",
+            "type": "shell",
+            "command": "python build.py",
```

Comment:
This might not be cross-compatible. On Windows is very common to run python using `py` instead of `python` (is actually a python launcher where you can select the python version if you have several installed).

Is there a way to configure the commands per operating system?

Reply:
There is a way to do it per OS, yeah. Will be easy set up.